apps/server/src/shared/infra/collaborative-storage/strategy/nextcloud/nextcloud.client.ts

```diff
@@ -55,7 +55,9 @@ export class NextcloudClient {
 			request,
 			(data: NextcloudGroups) => data.groups[0],
 			(error) => {
-				throw new NotFoundException(error, `Group ${groupName} not found in Nextcloud!`);
+				throw new NotFoundException(`Group ${groupName} not found in Nextcloud!`, {
+					cause: error instanceof Error ? error : undefined,
```

Comment:
cause allow undefined or error as paramter. You can write
throw new NotFoundException(`Group ${groupName} not found in Nextcloud!`, { cause: error })

Reply:
No, you can't because `error` has type `unknown`.